manifests/logical_volume.pp

```diff
@@ -11,6 +11,7 @@
   $fs_type           = 'ext4',
   $mkfs_options      = undef,
   $mountpath         = "/${name}",
+  $lvm_device_path   = "/dev/${volume_group}/${name}",
```

Comment:
I believe `$volume_group` will be unset at the time of parameter defaults as only `$name` is available in defaults. Could you migrate this to the $_volume_group pattern to make it work? Also, as no tests failed this is apparently not tested so a test would be great :).

The readme should mention the new parameter too. Thanks!